Task7/Task7/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task7
+{
+    //Каждое из чисел m и n нечетное. 
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите два числа");
+ 
+            int m = int.Parse(Console.ReadLine());
+            int n = int.Parse(Console.ReadLine());
+            Console.WriteLine(EvenOdd(m, n));
+
+        }
```

Comment:
Здесь анализ значения логического выражения.
Пустая строка перед } не нужна. Она только затрудняет чтение кода.
```suggestion
            if(IsBothNumbersOdd(m, n))
                Console.WriteLIne("Каждое из чисел нечетное.");
            else
                Console.WriteLIne("Не каждое из чисел нечетное.");
        }
```